webapp/src/pages/reviewDetails/index.js

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import ReactTable from "react-table";
+import { withRouter } from "react-router";
+import "react-table/react-table.css";
+
+import { reviewService } from "../../services/reviews";
+import { withTranslation } from 'react-i18next'
+
+class ReviewDetailsPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            infoColumns: [],
+            isLoading: true,
+            reviewDetails: [],
+        }
+        this.identifierCell = this.identifierCell.bind(this);
+        this.responseIDCell = this.responseIDCell.bind(this);
+        this.scoreCell = this.scoreCell.bind(this);
+    }
+
+    componentDidMount() {
+        this.loadReviewDetails();
+    }
+
+    responseIDCell = props => {
+        let reviewLink = `/${this.props.event.key}/review/${props.value}`
```

Comment:
This isn't being modified, so should be `const`.

Reply:
Agreed. Thanks!